docs/examples.rst

```diff
@@ -471,3 +471,42 @@ like so:
 .. include::  ../tests/core/contracts/test_contract_example.py
     :code: python
     :start-line: 1
+    
+Using Infura Rinkeby Node
+-------------------------
+Import your required libraries
+
+.. code-block:: python
+
+    from web3 import Web3, HTTPProvider
+    
+Initialize a web3 instance with an Infura node
+
+.. code-block:: python
+    
+    w3 = Web3(Web3.HTTPProvider("https://rinkeby.infura.io/v3/YOUR_INFURA_KEY"))
+
+ 
+Inject the middleware into the middleware onion
+
+.. code-block:: python
+    
+    w3.middleware_onion.inject(geth_poa_middleware, layer=0)
+    
+Just remember that you have to sign all transactions locally, as infura does not handle any keys from your wallet
+
+.. code-block:: python
+    
+    transaction = contract.functions.function_Name(params).buildTransaction()
```

Comment:
I think it would be helpful to either link to the "sign a transaction" docs (https://web3py.readthedocs.io/en/stable/web3.eth.account.html#sign-a-transaction) or add the lines that you have in the PR description: 
```
transaction = contract.functions.function_Name(params).buildTransaction()
transaction.update({ 'gas' : appropriate_gas_amount })  
transaction.update({ 'nonce' : web3.eth.getTransactionCount('Your_Wallet_Address') })
signed_tx = w3.eth.account.signTransaction(transaction, private_key)
```